src/export/StructureDefinitionExporter.ts

```diff
@@ -493,6 +502,64 @@ export class StructureDefinitionExporter implements Fishable {
     return this.fisher.fishForMetadata(item, ...types);
   }
 
+  applyElementProfiles(caretRules: CaretValueRule[], structDef: StructureDefinition): void {
+    const profileElementExtension =
+      'http://hl7.org/fhir/StructureDefinition/elementdefinition-profile-element';
+
+    caretRules.forEach(rule => {
+      // If the rule isn't applied to an element, it's not relevant
+      if (rule.path) {
```

Comment:
It could save you a level of nesting to just do something like:
```javascript
if (!rule.path) {
  return;
}
```
that way the rest of the code does not have to be nested at that level